react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,35 +1,24 @@
-import React, { Component } from 'react';
+import React from 'react';
 
 import Square from '../Square';
 
 import style from './style.scss';
 
-class Board extends Component {
-  renderSquare(i) {
-    return <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
-  }
+function Board({ squares, onClick }) {
+  const renderSquare = squarePosition => (
+    <Square value={squares[squarePosition]} onClick={() => onClick(squarePosition)} />
+  );
+  const row1 = [0, 1, 2];
+  const row2 = [3, 4, 5];
+  const row3 = [6, 7, 8];
```

Comment:
These rows will always be the same. How about declaring them outside the function, as to avoid the constant declaration everytime this function is used